docs.go

```diff
@@ -119,6 +121,40 @@ Workflow options:
     -f --noconfirm
      Do not prompt confirmation dialog.
 
+  -M --maintenances
+    Search and operate on configuration of maintenance.
+    Maintenance could be filtered using /<pattern> argument, for example,
+    search maintenance match the word 'update-kernel':
+      zabbixctl -M dbnode-* /update-kernel
+
+    -a --add
+      Add new specified <maintenance> with timeperiod type once.
+
+    --start <date>
+      Start date 'yyyy-mm-dd HH:MM'. Default now.
+
+    --end <date>
+      Stop date 'yyyy-mm-dd HH:MM'. Default now + period.
+
+    --period <date>
+      Period in m/h/d (minutes/hours/days).
+      [default: 1d]
+
+    -f --noconfirm
+      Do not prompt confirmation dialog.
+
+    -r --remove
+      Remove specified <maintenance>.
+
+    -z --read-stdin
+      Read hosts from stdin.
+
+  -H --hosts
+    Search and operate with hosts.
+
+    -r --remove
+      Remove hosts.
```

Comment:
This description doesn't so clear to the understanding. It's a boolean argument?

Reply:
It's string, fixed description.